spec/lib/submission_builder/return_header1040_spec.rb

```diff
@@ -359,17 +359,35 @@
     context "efile security information" do
       context "UserAgentTxt" do
         before do
-          submission.client.efile_security_informations.first.update(user_agent: "Mozilla/5.0 (Linux; Android 10; SAMSUNG SM-S205DL) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/12.1 Chrome/79.0.3945.136 Mobile Safari/537.36")
-          submission.client.efile_security_informations.last.update(user_agent: "Mozilla/5.0 (Linux; Android 10; SAMSUNG SM-S205DL) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/12.1 Chrome/79.0.3945.136 Mobile Safari/537.36")
+
+          submission.client.efile_security_informations.first.update(
+              user_agent: "Mozilla/5.0 (Linux; Android 10; SAMSUNG SM-S205DL) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/12.1 Chrome/79.0.3945.136 Mobile Safari/537.36",
+              browser_language: "es-419",
+              platform: "Linux armv8l-more"
+          )
+          submission.client.efile_security_informations.last.update(
+              user_agent: "Mozilla/5.0 (Linux; Android 10; SAMSUNG SM-S205DL) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/12.1 Chrome/79.0.3945.136 Mobile Safari/537.36",
+              browser_language: "es-419",
+              platform: "Linux armv8l-more"
+          )
         end
 
         it "trims long user agent text down to 150 characters, the max acceptable by the schema" do
           expect(submission.client.efile_security_informations.first.user_agent.length).to eq 151
           expect(submission.client.efile_security_informations.last.user_agent.length).to eq 151
+          expect(submission.client.efile_security_informations.first.browser_language.length).to eq 6
+          expect(submission.client.efile_security_informations.last.browser_language.length).to eq 6
+          expect(submission.client.efile_security_informations.last.platform.length).to eq 17
+          expect(submission.client.efile_security_informations.last.platform.length).to eq 17
+
           response = SubmissionBuilder::ReturnHeader1040.build(submission)
           xml = Nokogiri::XML::Document.parse(response.document.to_xml)
           expect(xml.at("FilingSecurityInformation AtSubmissionFilingGrp UserAgentTxt").text.length).to eq 150
           expect(xml.at("FilingSecurityInformation AtSubmissionCreationGrp UserAgentTxt").text.length).to eq 150
+          expect(xml.at("FilingSecurityInformation AtSubmissionFilingGrp BrowserLanguageTxt").text.length).to eq 5
```

Comment:
I would prefer that this validate the truncated string, which I don't see. I also can live with it as-is.